lib/cube_browser/__init__.py

```diff
@@ -539,26 +539,33 @@ def __init__(self, plots):
 
         self._slider_by_name = {}
         self._name_by_slider_id = {}
+        if self._axis_by_name:
+            name_size = max([len(name) for name in self._axis_by_name])
```

Comment:
It might be a little pedantic, but I wonder whether `name_len` would work better for this variable name? I mean, `name_size` does make you wonder whether the names will be
# huge 
or
### not so huge

...